component/argocd.jsonnet

```diff
@@ -351,9 +353,44 @@ local ssh_secret = kube._Object('v1', 'Secret', 'argo-ssh-key') {
   type: 'Opaque',
 };
 
+
+// Manually adding certificate for conversion webhook
+// as the upstream kustomize is broken.
+// 2023/02/19 sfe
+local webhook_certs = [
```

Comment:
This will most likely completely break cluster bootstrapping because ArgoCD won't be able to sync itself when cert-manager is missing, and cert-manager isn't necessarily enabled on all clusters and won't get installed even if the component is enabled because the cert-manager app doesn't sync in the bootstrap argocd since it needs secrets from Vault for the vshn-acme-dns integration.

Reply:
True 🤔 I will come up with something

Reply:
This upgrgade is a mess.

Reply:
The conversion webhook is now configurable (enable/disable).  

Marked this change as breaking and added an upgrade how-to.